forloop_modules/globals/local_variable_handler.py

```diff
@@ -200,10 +200,11 @@ def create_variable(self, name, value, is_result: Optional[bool] = None, additio
         elif self.handler_mode == "variable":
             ncrb_fn = ncrb.new_variable
 
+        optional_args = {"is_result": is_result} if is_result is not None else {}
```

Comment:
Could we change this somehow to either remove it from the structure or add it as obligatory? I think this way it can be a source of a lot of potential future problems